src/library/psi_library.jl

```diff
@@ -1149,3 +1149,415 @@ function build_two_zone_5_bus(; kwargs...)
     end
     return sys
 end
+
+const COST_PERTURBATION_NOISE = rand(1_000_000)
```

Comment:
Does this need to be in a function so that it is guaranteed to be regenerated every time you start julia?